mappings/net/minecraft/world/gen/feature/FancyTreeFeature.mapping

```diff
@@ -0,0 +1,3 @@
+CLASS bry net/minecraft/world/gen/feature/FancyTreeFeature
```

Comment:
Fancy? Is there a better name for this? Does it apply to large trees or what?

Reply:
It's the large oak, yeah. Again, shitty Mojang names but it feels weird to change it, it's like changing a Block's name

Reply:
Why not just LargeTreeFeature?

Reply:
Large seems a lot better imo